__tests__/nbaUtilsFunctionTest.js

```diff
@@ -115,4 +115,17 @@ describe('../util/nbaUtils', ()=>{
         received[5] + "\nSunday: " + received[6]);
         expect(received).toEqual(expected);
     });
-})
\ No newline at end of file
+
+    test('getAllPlayerNames() returns list of player names', async ()=>{
+        const leagueId = 194346
+        const teamId = Math.floor(Math.random() * 10) + 1 // Random number between [1, 10]
+        const accessToken = "" // WIP need way to easily get access token
+        const rawRoster = await nbaData.getPlayerRoster(accessToken, leagueId, teamId).then(resp=>(resp));
+
+        let players = nbaUtils.getAllPlayerNames(rawRoster);
+        players.forEach(received => {
+            console.log("Expected: <Object>\nReceived: " + received);
```

Comment:
I think we want to check if at least the first and last name fields in the JSON object are not empty.
This only checks if a JSON object exists which it will because you push it in the array.

Reply:
Sure, I didn't know how or what to validate anyways.